test/fixtures/full-static-with-ignore-env/full-static-with-ignore-env.test.js

```diff
@@ -0,0 +1,3 @@
+import { buildFixture } from '../../utils/build'
```

Comment:
I might be missing something, but is this used to test the feature?

Reply:
To be fully honest I wasn't quite sure if this was the right way to test the feature using the suggested `ignoreEnv` option in `nuxt.config.js`.